frontend/src/components/ChannelClients.vue

```diff
@@ -105,6 +131,15 @@ export default {
     }
   },
   methods: {
+    async poke() {
+      //this.pokeClientDialog = true;
+      await this.$TeamSpeak.execute('clientpoke', {
+        msg: this.pokeMessage,
+        clid: this.client.clid
+      });
```

Comment:
Looks good to me. The executed command needs to be wrapped by a try catch block for error handling.  
```suggestion
      try {
        await this.$TeamSpeak.execute('clientpoke', {
          msg: this.pokeMessage,
          clid: this.client.clid
        });
      } catch(err) {
        this.$toast.error(err.message)
      }
```

Reply:
awesome! thank you